docs/self_hosting.md

```diff
@@ -43,6 +43,9 @@ Finally, unpack the `dist/threema-web-[VERSION].tar.gz` archive to your web serv
 Make sure to serve Threema Web only via https. We also recommend to enable
 HSTS, HPKP, CSP and other available security mechanisms in your web server.
 
+#### Notes on building on macOS
+Building on macOS works as well if you have gnu-sed installed. You can get gnu-sed from homebrew with ```brew install gnu-sed```.
```

Comment:
gnu-sed is not necessary with when not using `gsed`, right?

Reply:
Yes, I reverted it in https://github.com/threema-ch/threema-web/pull/406/commits/2c6d3e0a69c52039aa8be2885ae350882be44744